### PR TITLE
Jira DOC-983: Predis client connection SSL statement is out-of-date

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

File: content/rs/references/client_references/client_php.md [hunk 1]
```diff
@@ -78,8 +84,23 @@ To enable persistent connections, use the `persistent` connection attribute, as
         "persistent" => "1"));
 ```
 
-## SSL
+## Encrypted connections
+
+[As of v1.1.0](https://github.com/predis/predis/blob/main/CHANGELOG.md#v110-2016-06-02), you can encrypt connections specifying `tls` or `rediss` as the value of the `scheme` attribute or as part of the URI.
```
Comment (rrelledge) [inline content/rs/references/client_references/client_php.md:89]:
```suggestion
[As of v1.1.0](https://github.com/predis/predis/blob/main/CHANGELOG.md#v110-2016-06-02), you can encrypt connections by specifying `tls` or `rediss` as the value of the `scheme` attribute or as part of the URI.
```

File: content/rs/references/client_references/client_php.md [hunk 2]
```diff
@@ -59,9 +63,11 @@ var_dump($value);
...
-    $ php predis_example.php
-    Connected to Redis
-    string(3) "bar"
+```console
+$ php predis_example.php
+Connected to Redis
+string(3) "bar"
+```
 
 ## Persistent connections
```
Comment (rrelledge) [inline content/rs/references/client_references/client_php.md:72]: For the example code under Persistent connections, you might want to remove one level of indentation from the code block to make the text align left.

**Review by rrelledge — APPROVED** [review]
Looks good to me.

File: content/rs/references/client_references/client_php.md (hunk 2 above)
Comment (lanceleonard) [reply content/rs/references/client_references/client_php.md:72]: Fixed.

File: content/rs/references/client_references/client_php.md (hunk 1 above)
Comment (lanceleonard) [reply content/rs/references/client_references/client_php.md:89]: Fixed.

**Pull request merged**